src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractLanguageService`2.VsCodeWindowManager.cs

```diff
@@ -107,6 +139,21 @@ private void AddOrRemoveDropdown(bool enabled)
                 {
                     RemoveDropdownBar(dropdownManager);
                 }
+
+                static bool IsOurDropdownBar(IVsDropdownBarManager dropdownBarManager, IVsDropdownBarClient? dropdownBarClient, out IVsDropdownBar? existingDropdownBar)
```

Comment:
Why the static here? Then it means we're having to pass down the _dropdownBarClient which makes the flow a bit more confusing.